samples/RazorLight.Samples/Samples.EntityFrameworkProject.csproj

```diff
@@ -1,29 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
netcoreapp2.1 is not out of LTS. See: https://dotnet.microsoft.com/platform/support/policy/dotnet-core It expires August 2021.  Please don't remove this as it basically shows people how to use the project in supported versions of .NET Core.  You can remove .netcoreapp3.0